modules/base/templates/backups/wikitide-backup.py.erb

```diff
@@ -258,14 +297,16 @@ def download(source: str, dt: str, database: str):
     ts = time.time()
     print(f'Downloading backup of \'{source}\' for date {dt}...')
 
-    if source in ['private', 'sslkeys', 'phorge', 'swift-account-container']:
+    if source in ['private', 'sslkeys', 'phorge', 'swift-account-container', 'mattermost-data']:
         download_pca(f'{source}/{dt}.tar.gz')
     elif source in ['mediawiki-xml']:
         download_pca(f'{source}/{database}/{dt}.xml.gz')
     elif source in ['grafana']:
         download_pca(f'sql/{source}/{dt}.tar.gz')
     elif source in ['sql']:
         download_pca(f'{source}/{database}/{dt}.sql.gz')
+    elif source in ['mattermost-db', 'puppetdb']:
+        download_pca(f'{source}/{dt}.sql.gz')
```

Comment:
Is there a reason we don't use the same format as grafana above?

Reply:
Yes, because we don't use:

```
    tar = tarfile.open('grafana.tar.gz', 'w:gz')
    tar.add('/var/lib/grafana/grafana_backup.db', arcname='grafana_backup.db')
    tar.close()
```